gbpservice/nfp/orchestrator/drivers/orchestration_driver.py

```diff
@@ -36,7 +37,10 @@
 
 def _set_network_handler(f):
     def wrapped(self, *args, **kwargs):
-        device_data = args[0]
+        if type(args[0]) == dict:
```

Comment:
Depending on data type to pick argument selectively is a risk. Can't this be done differently?